addons/test_discuss_full/tests/test_performance.py

```diff
@@ -12,7 +12,7 @@
 
 @tagged('post_install', '-at_install')
 class TestDiscussFullPerformance(HttpCase):
-    _query_count = 59
+    _query_count = 72
```

Comment:
Are you sure about this number? Wasn't it lowered at some point? 13 queries seems like a lot to just read `livechat_lang_ids`

Reply:
it was from rebase issue, i pushed the new number